src/Backend/Backend.ts

```diff
@@ -34,6 +36,10 @@ function backendRegistrationApi() {
       const backendName = backend.name();
       assert(backendName.length > 0);
       globalBackendMap[backendName] = backend;
+      const compiler = backend.compiler();
```

Comment:
I will change this `compiler` name to `toolchain` on next PR.
There are some patches implemented using this name, so I want to apply this code first and change the name later.

Reply:
👍